### PR TITLE
Update your PR to have tests pass

### PR DESCRIPTION
**Pull request by kshyatt**
Fixed the bug in conversion between CSC and CSR, stripped out a LOT of dead tests, things were passing locally (on windows!!!) for me. We have a LOT of library methods which we don't wrap - the new generic COO ones. Do we want to wrap those before we merge this or make a new PR?

### CODE REVIEW

**Comment by maleadt** [comment]
> We have a LOT of library methods which we don't wrap - the new generic COO ones. Do we want to wrap those before we merge this or make a new PR?

Not necessarily, having them available as low-level wrappers already improves the status quo.

bors try

File: lib/cusparse/wrappers.jl [hunk 1]
```diff
@@ -81,13 +81,13 @@ for (fname,elty) in ((:cusparseScsr2csc, :Float32),
                 # TODO: algorithm configuratibility?
                 @workspace size=@argout(
                         cusparseCsr2cscEx2_bufferSize(handle(), m, n, csr.nnz, csr.nzVal,
-                            csr.rowPtr, csr.colVal, nzVal, rowVal, colPtr,
```
Comment (maleadt) [inline lib/cusparse/wrappers.jl:84]: :frowning:

**Review by maleadt — APPROVED** [review]

File: lib/cusparse/wrappers.jl (hunk 1 above)
Comment (kshyatt) [reply lib/cusparse/wrappers.jl:84]: my reaction exactly

**Comment by kshyatt** [comment]
Whoops! I was running the `cublas` and `cusparse` tests only. Fixing `cusolver` now...

**Comment by kshyatt** [comment]
bors try

**Comment by kshyatt** [comment]
`csreigs` (which worked previously) is now giving this lovely message:

```
      From worker 2:    Please submit a bug report with steps to reproduce this fault, and any error messages that follow (in their entirety). Thanks.
      From worker 2:    Exception: EXCEPTION_ACCESS_VIOLATION at 0x7ffac9affb8c -- cusolverSpZcsreigvsiHost at C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\bin\cusolver64_10.DLL (unknown line)
      From worker 2:    in expression starting at C:\Users\kshyatt\.julia\dev\CUDA\test\cusolver\cusparse.jl:12
      From worker 2:    cusolverSpZcsreigvsiHost at C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\bin\cusolver64_10.DLL (unknown line)
      From worker 2:    cusolverSpZcsreigvsiHost at C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\bin\cusolver64_10.DLL (unknown line)
      From worker 2:    cusolverSpDcsreigsHost at C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\bin\cusolver64_10.DLL (unknown line)
      From worker 2:    macro expansion at C:\Users\kshyatt\.julia\dev\CUDA\lib\cusolver\libcusolver.jl:3931 [inlined]
      From worker 2:    macro expansion at C:\Users\kshyatt\.julia\dev\CUDA\src\memory.jl:312 [inlined]
      From worker 2:    macro expansion at C:\Users\kshyatt\.julia\dev\CUDA\lib\cusolver\error.jl:52 [inlined]
      From worker 2:    cusolverSpDcsreigsHost at C:\Users\kshyatt\.julia\dev\CUDA\lib\utils\call.jl:93
      From worker 2:    csreigs at C:\Users\kshyatt\.julia\dev\CUDA\lib\cusolver\wrappers.jl:242

```

**Comment by kshyatt** [comment]
bors try

**Comment by kshyatt** [comment]
Let's try again with the new commit...

**Comment by kshyatt** [comment]
bors try

**Comment by kshyatt** [comment]
Not sure I formatted the patch right, feel free to push to fix it.

**Comment by kshyatt** [comment]
bors try

**Pull request merged**